client/src/components/Sidebar/Sidebar.js

```diff
@@ -2,13 +2,53 @@ import React from 'react';
 import { Nav, NavItem, NavLink } from 'reactstrap';
 import { NavLink as RRNavLink, useRouteMatch } from 'react-router-dom';
 import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Icon } from '_common';
 import * as ROUTES from '../../constants/routes';
 import HistoryBadge from '../History/HistoryBadge';
 import './Sidebar.global.scss'; // XXX: Global stylesheet imported in component
 import './Sidebar.module.scss';
 
+/** Navigation list item **/
+const SidebarItem = ({ to, label, iconName, children, disabled }) => {
```

Comment:
__Kudos__: Great to see. I hope this can later be promoted to a common component, that any Sidebar (main or section-specific) can use.